airflow/models/taskinstance.py

```diff
@@ -3071,6 +3053,11 @@ def _execute_task(self, context: Context, task_orig: Operator):
         """
         return _execute_task(self, context, task_orig)
 
+    def update_last_heartbeat(self):
```

Comment:
yeah here too doesn't `update_heartbeat()` or `heartbeat()` sound better?

Reply:
update_heartbeat makes sense here (I didn't want to use just `heartbeat` as that does more on the `Job` subclasses and I wanted to differentiate that it was just updating a column and nothing more.)